src/api/client.tsx

```diff
@@ -86,6 +86,7 @@ function createGQLClient(config: Config): Client {
             // See: https://formidable.com/open-source/urql/docs/graphcache/normalized-caching/#custom-keys-and-non-keyable-entities
             keys: {
                 ApplicationConfiguration: () => null,
+                EventQuotePriceInfo: () => null,
```

Comment:
Yeah, it silences a warning about using a gql field without an id key.

For example...

```
myAwesomeField {
    name
    title
}
```

Since not all fields that are returned as objects have an actual `id`, we need to let the urql gql client know which of those fields we expect not to have an `id`, otherwise the warning will show up to make sure you didn't just forget it.